doc/en/user/source/data/app-schema/solr-tutorial.rst

```diff
@@ -93,4 +93,115 @@ The complete mapping file is:
       </typeMappings>
   </as:AppSchemaDataAccess>
 
-The mappings for the attributes are straightforward and follow the normal App-Schema attributes mappings syntax. Currently multi valued fields are not supported.
\ No newline at end of file
+The mappings for the attributes are straightforward and follow the normal App-Schema attributes mappings syntax. Currently multi valued fields are not supported.
+
+Using Solr as App-Schema Indexes
+--------------------------------
+
+App-Schema Indexes is an extension for mapping that allows to use Apache Solr as Index for queries and retrieving data from normal App-Schema datasource (SQL DB, MongoDB, ... ).
+
+The only requirement to use it is having Geoserver App-Schema extension and Solr extension installed.
+
+How Index layer works
+^^^^^^^^^^^^^^^^^^^^^
+
+When App-Schema detects the index layer is activated for a FeatureType, it will use Solr configured fields for every query incoming from Geoserver OWS requests.  If the incoming query uses only indexed fields App-Schema will query only on Solr data source for retrieving matching features IDs and will connect to normal data source to get all in depth data but exclusively for matching IDs.
+
+.. warning:: note that both Primary Keys (solr index core and data source) should match to get Index layer working. 
+
+Linking an index only store
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Begin creating the SolrDataStore definition as usual along with the Postgis store definition:
```

Comment:
This section need to be reviewed, since this is a tutorial I would include the full mappings file.